storage/store_pool.go

```diff
@@ -39,15 +42,24 @@ const (
 	// TestTimeUntilStoreDeadOff is the test value for TimeUntilStoreDead that
 	// prevents the store pool from marking stores as dead.
 	TestTimeUntilStoreDeadOff = 24 * time.Hour
+
+	// timeoutForDeclinedReservations is the amount of time to not consider the
+	// store available for up-replication after a reservation was declined.
+	timeoutForDeclinedReservations = 10 * time.Second
```

Comment:
Yep.  I'm splitting it into two values.  10 seconds for failures, 1 second for denials.

PR is forthcoming.